src/ProjectTempoListener.cpp

```diff
@@ -53,4 +59,15 @@ void ProjectTempoListener::OnProjectTempoChange(double newTempo)
 {
    for (auto track : mTrackList)
       track->OnProjectTempoChange(newTempo);
+   
+   auto& viewInfo = ViewInfo::Get(mProject);
+   if(!viewInfo.playRegion.Empty() && mTempo > 0 && newTempo > 0)
```

Comment:
Can `mTempo` possibly still be zero?

Reply:
I don't think that this is the possible case right now, but won't harm to check that. Non-positive tempo doesn't make sense, right?

Reply:
Sure, I just wanted to know if you were aware of a situation where it wasn't.